### ISSUE TITLE
.prefix remains actionable by NadekoBot even if the prefix is changed

### ISSUE THREAD

**Issue by bender-the-greatest** [Kwoth/NadekoBot#2481]
In our community, I've changed the NadekoBot command prefix to be `!` instead of `.`. Every command so far, including `!prefix` works as expected, but when I type `.prefix` it continues to be processed by NadekoBot. Since the main reason of changing the prefix is to uniquify commands that may be implemented by multiple bots, I would expect that `.prefix` shouldn't be processed by NadekoBot when the prefix is something else.

Basically what I'm trying to do here is change the prefix to `!` temporarily so I can install another bot (which uses `.` as its default prefix, but can be changed with its own `.prefix` command), change its prefix to something else, then change NadekoBot back to use a `.` prefix.

**Comment by Kaoticz** [comment]
This is not a bug, it's a feature. It's there in case someone forgets what's the currently set prefix in place (I've seen it happening several times).

Here's what you can do: create a channel your new bot can see but not Nadeko and change its prefix in there. Should work just fine.

Once you're done, please, close this ticket.

**Comment by bender-the-greatest** [comment]
Would it not be better to have something like another documented command called something like `.checknadekoprefix` (`.chprefix`) that is always actionable at the default prefix, rather than ignoring the override on a command that is commonly overloaded by other bots?

I got what I needed working by turning off Nadeko while I set up the other bot, but strongly feel the workaround should not be necessary. Your workaround won't work without additional commands to disable nadeko actions on the "set up channel" if Nadeko is an administrator on a server and can inherently see every channel.
